docs/content/preview/architecture/key-concepts.md

```diff
@@ -145,7 +168,7 @@ Tablets are also referred as shards.
 
 ## Tablet leader
 
-In a cluster, each [tablet](#tablet) is replicated as per the [replication factor](#replication-factor-rf) for high availability. Amongst these tablet replicas one tablet is elected as the leader and is responsible for handling writes and consistent reads. The other replicas as termed followers.
+In a cluster, each [tablet](#tablet) is replicated as per the [replication factor](#replication-factor-rf) for high availability. Amongst these tablet replicas one tablet is elected as the leader and is responsible for handling writes and consistent reads. The other replicas are called followers.
```

Comment:
I think we should either add a separate entry for tablet followers or change the title to Tablet leader and followers.